cm.py

```diff
@@ -315,6 +315,14 @@ def deploy_cluster_kerberos_client_config(self, cluster_name, host_ids=None):
         }
         return self.api_client.deploy_cluster_kerberos_client_config(cluster_name, host_ref_list)
 
+    def get_cm_kerberos_principals(self):
+        """Get list of Kerberos principals needed by the services being managed by Cloudera Manager.
```

Comment:
A slightly redundant docstring :) Might wanna just have the returns be `A list of strings` or whatever.

Reply:
Done.